lib/arch/ssse3/dec_reshuffle.c

```diff
@@ -1,24 +1,32 @@
-static inline __m128i
-dec_reshuffle (__m128i in)
-{
-	// Mask in a single byte per shift:
-	const __m128i maskB2 = _mm_set1_epi32(0x003F0000);
-	const __m128i maskB1 = _mm_set1_epi32(0x00003F00);
+static inline __m128i dec_reshuffle(__m128i in) {
+	// in
+	// 00llllll 00kkkkLL 00jjKKKK 00JJJJJJ
+	// 00iiiiii 00hhhhII 00ggHHHH 00GGGGGG
+	// 00ffffff 00eeeeFF 00ddEEEE 00DDDDDD
+	// 00cccccc 00bbbbCC 00aaBBBB 00AAAAAA
```

Comment:
I understand what the lower/upper case notation means now, but it could really do with a friendly explanation.